examples/multiedgetracking/feature_extraction/worker.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+import cv2
+import torch
+import torchvision.transforms as T
+from PIL import Image
+from typing import List
+
+from functools import reduce
+from threading import Thread
+
+from sedna.common.log import LOGGER
+from sedna.core.multi_edge_tracking.utils import get_parameters
+from sedna.core.multi_edge_tracking.plugins import PluggableModel
+from sedna.core.multi_edge_tracking.data_classes import DetTrackResult, Target
+from sedna.core.multi_edge_tracking.components.feature_extraction import FEService
```

Comment:
The cooperation between this `FEService` interface and other interfaces in Multi-Edge Tracking needs to be clearly explained, just like the cooperation between large model and small model in joint inference.